serverless.yaml

```diff
@@ -520,6 +520,12 @@ resources:
       FHIRBinaryBucket:
         Description: S3 Bucket for storing Binary Objects
         Value: !Ref FHIRBinaryBucket
+      ResourceDynamoDBTableV2Arn:
+        Description: DynamoDB table for storing non-Binary resources
+        Value: !GetAtt ResourceDynamoDBTableV2.Arn 

```

Comment:
From the email, it sounds like they want us to export the `StreamArn` as well

Reply:
Yep I think you are right